examples/openui5/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en" data-framework="openui5">
+
+<head>
+	<meta charset="utf-8">
+	<title>OpenUI5 • TodoMVC</title>
+	<script id="sap-ui-bootstrap" src="https://openui5.hana.ondemand.com/resources/sap-ui-core.js" data-sap-ui-theme="sap_bluecrystal" data-sap-ui-libs="sap.m" data-sap-ui-xx-supportedLanguages="en" data-sap-ui-resourceroots='{"ToDoMVC": "./js"}'></script>
```

Comment:
Unfortunately, that doesn't work. This file initiates lazy loading of many further files, depending on what components the app actually uses. The only way to have them files in there is to put the whole library of JS files in the commit.. That blows up the size though (as I figured after the initial PR).